fares-data-build-tool/create-data
# feat: Netex changes for exempt stops and Hybrids with exemptions

## Description

Altering netex converter so it adds a section for exempt stops, if the user's ticket has some. Also added logic for hybrid tickets which can have both exempt stops and exempt services.

## Testing instructions

All netex in repo currently validates - we need to check this on the test environment to make sure the matching JSON we create works in the netex repo (which means we need to export).
